src/core/server/http/router/request.ts

```diff
@@ -91,40 +90,27 @@ export class KibanaRequest<
    * received in the route handler.
    * @internal
    */
-  private static validate<
-    P extends ObjectType,
-    Q extends ObjectType,
-    B extends ObjectType | Type<Buffer> | Type<Stream>
-  >(
+  private static validate<P, Q, B>(
     req: Request,
-    routeSchemas: RouteSchemas<P, Q, B> | undefined
+    routeSchemas: RouteValidator<P, Q, B> | undefined
```

Comment:
nit: rename `routeSchemas` to `routeValidator` and maybe make optional with `?` instead of `| undefined`?

Reply:
It makes a difference, `?` allows you not to send the parameter. `| undefined` enforces you to send the parameter, but it accepts the parameter to be `undefined`. This way you don't forget to send it by mistake ;)

Nevermind, I found a better approach so we always can send the RouteValidator and we can remove the `if undefined` in it.